pkg/registry/apis/provisioning/register.go

```diff
@@ -42,61 +43,77 @@ func RegisterAPIService(
 }
 
 func (b *ProvisioningAPIBuilder) GetAuthorizer() authorizer.Authorizer {
-	return nil // default authorizer is fine
+	return authorizer.AuthorizerFunc(
+		func(ctx context.Context, a authorizer.Attributes) (authorizer.Decision, string, error) {
+			if a.GetSubresource() == "webhook" {
+				// for now????
```

Comment:
we can verify GH webhook deliveries. Perhaps worth adding a link to that in the comments: https://docs.github.com/en/webhooks/using-webhooks/validating-webhook-deliveries

Reply:
yes, here is the example: https://github.com/grafana/grafana/blob/spike/git-sync-ui-webhook-implementation/pkg/services/github/service.go#L288-L292

I will port them there after we merge the stubs.

Reply:
The k8s authorizer does not have the payload -- it only has the user + requested path.   I don't think github can send a valid user, so we have to allow everything and then do real validation (in @MissingRoberto 's branch) inside the http.Handler